examples/mobile/HomeApp/js/app.js

```diff
@@ -101,8 +101,10 @@
 	//TODO: provide mechanism for getting web clip name from webClip meta data
 	//		and separate from getting ID
 	function getWebClipName(webClip) {
-		webClip = webClip.replace("webclip/", "").replace("/", "");
-		return webClip;
+		const pos = webClip.lastIndexOf("/"),
```

Comment:
maybe regexp  will be better `.replace(/\/$/, "")`

Reply:
This is working properly
`/.*\//`
https://regex101.com/r/CRiluw/2